components/pages/privacy/resourceAccess/revokeAccessButton/index.jsx

```diff
@@ -144,7 +144,9 @@ export default function RevokeAccessButton({
   const podRoot = usePodRootUri(session.info.webId);
   const policiesContainer = podRoot && getPoliciesContainerUrl(podRoot);
   const resourceName =
-    resources.length === 1 ? getResourceName(resources[0]) : "your pod";
+    resources.length === 1
+      ? getResourceName(resources[0])
+      : "anything in your pod";
```

Comment:
It'd be nice to have all of these strings as constants - that will make it easier to swap them out for localized strings later.